projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/strategy/datajob/JobFieldStrategyBy.java

```diff
@@ -29,7 +29,8 @@
    DESCRIPTION("config.description", "content/config/description"),
    SOURCE_URL("config.sourceUrl", "content/config/sourceUrl"),
    NEXT_RUN_EPOCH_SECS("config.schedule.nextRunEpochSeconds", "content/config/schedule/nextRunEpochSeconds"),
-   SCHEDULE_CRON("config.schedule.scheduleCron", "content/config/schedule/scheduleCron");
+   SCHEDULE_CRON("config.schedule.scheduleCron", "content/config/schedule/scheduleCron"),
+   DATA_JOB_CONTACTS("config.contacts", "content/config/contacts");
```

Comment:
What do you think about `config.contacts.enabled`?

Reply:
Maybe config.contacts.present would be a more accurate name?

Reply:
`config.contacts.present` will work.

Reply:
changed in latest commit.